plugins/pv_lxc.c

```diff
@@ -344,28 +340,6 @@ static void pv_setup_lxc_container(struct lxc_container *c,
 		sprintf(entry, "pv-unknown");
 	c->set_container_type(c, entry);
 
-	/*
-	 * Set console filename if not provided.
-	 */
-	if (pv_lxc_capture_logs_activated()) {
-		memset(entry, 0, sizeof(entry));
-		c->get_config_item(c, "lxc.console.logfile", entry,
-				   sizeof(entry));
-		if (!strlen(entry)) {
-			__pv_paths_pv_log_file(path, PATH_MAX,
-					       __pv_get_instance()->state->rev,
-					       c->name, LXC_CONSOLE_LOG_FNAME);
-			c->set_config_item(c, "lxc.console.logfile", path);
-		}
-	}
-	/*
-	 * Put a hard limit of 2MiB on console file size if one is not defined.
-	 */
-	if (c->get_config_item(c, "lxc.console.size", NULL, 0)) {
```

Comment:
if we stop supporting these we need to change config.h and config.c too.... not sure if there is a smart way to honour these in the new world.... if so doing that of course would be amazing if not too much work...

Reply:
Right now, logserver uses a constant called MAX_SIZE (lines 155 and 210) to truncate the logs (all generated logs) I know that is not the ideal situation but at least there is no risk of using all the memory. And you are right, we need to support that configuration and I think the better way is inside the logserver.

Reply:
ok, lets chat about what the current configuration options are on the log front... my memory needs to be reminded i guess as its been a while ...